wallet-new/server/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -124,21 +125,20 @@ legacyWalletBackend WalletBackendParams {..} ntpStatus =
 -- TODO: no web socket support in the new wallet for now
 walletBackend :: (HasConfigurations, HasCompileInfo)
               => NewWalletBackendParams
-              -> Kernel.PassiveWallet
               -> Plugin Kernel.WalletMode
-walletBackend (NewWalletBackendParams WalletBackendParams{..}) passive =
+walletBackend (NewWalletBackendParams WalletBackendParams{..}) =
     first one $ worker walletServerOuts $ \diffusion -> do
       env <- ask
       let diffusion' = Kernel.fromDiffusion (lower env) diffusion
-      Kernel.bracketActiveWallet passive diffusion' $ \active ->
+      bracketActiveWallet diffusion' $ \active ->
```

Comment:
It is important that we initialize the passive wallet only once, so it's not correct to re-initialize it when we initialize the active wallet. That `passive` argument should not be removed here.